<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konkurs krzykaczy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Wybierz urządzenie do nagrywania</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="deviceComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="recordButton">
       <property name="text">
        <string>Nagrywaj</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="AddUserButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj użytkownika</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="AdminUserList">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>751</width>
      <height>371</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="EditUserButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edytuj użytkownika</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Wyświetlanie</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="AllRadioButton">
       <property name="text">
        <string>Wszyscy</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="MenRadioButton">
       <property name="text">
        <string>Mężczyźni</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="WomenRadioButton">
       <property name="text">
        <string>Kobiety</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuT">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionExportToCsv"/>
    <addaction name="actionImportFromCsv"/>
    <addaction name="separator"/>
    <addaction name="actionCalibrate"/>
    <addaction name="actionCalibrateFromFile"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuT"/>
  </widget>
  <action name="actionExportToCsv">
   <property name="text">
    <string>Eksportuj do CSV...</string>
   </property>
  </action>
  <action name="actionImportFromCsv">
   <property name="text">
    <string>Importuj z CSV...</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Kalibruj</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Zakończ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCalibrateFromFile">
   <property name="text">
    <string>Kalibruj z pliku</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
